### ISSUE TITLE
NTR request?

### ISSUE THREAD

**Issue by KristenKC** [microbialphenotypes/OMP-ontology#195]
Should we create child terms for pilus production regarding curli production? Is there a better name for this term?

**Comment by dsiegele** [comment]
Yes, OMP should have terms for pilus production phenotypes. I will change the name of OMP:0000084 ! pilus phenotype, which is a grandchild of cell morphology phenotype, to pilus morphology phenotype. 

Is a change in the number of pili a pilus production or pilus morphology phenotype? Currently, increased and decreased number of pili are in the cell morphology branch.
